### ISSUE TITLE
Noverlap Layout Worker: Uncaught ReferenceError: setImmediate is not defined

### ISSUE THREAD

**Issue by tom-osint** [graphology/graphology#358]
```
worker.js:68 Uncaught ReferenceError: setImmediate is not defined
    at r.NoverlapLayoutSupervisor.handleAddition (worker.js:68:1)
    at r.__webpack_modules__../node_modules/graphology/dist/graphology.umd.min.js._.emit (graphology.umd.min.js:1:6887)
    at graphology.umd.min.js:1:53640
    at r.i.addNode (graphology.umd.min.js:1:53708)
    at index.js:183:1
    at Array.forEach (<anonymous>)
    at OCGraph.compileNodes (index.js:183:1)
    at <anonymous>:1:7
```

**Comment by Yomguithereal** [comment]
Thanks @tom-osint, I always forget that `setImmediate` is not standard. A fix has been released in v0.4.2.

**Comment by tom-osint** [comment]
> Thanks @tom-osint, I always forget that `setImmediate` is not standard. A fix has been released in v0.4.2.

Cheers, appreciate it.
